tests/desktop/test_switch_to_mobile_site.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from unittestzero import Assert
+from pages.desktop.base import Base
+from pages.desktop.page_provider import PageProvider
+from pages.mobile.page_provider import MobilePageProvider
```

Comment:
You can leave this in `pages.mobile.page_provider` as `PageProvider` and then, in this file just do:

`from pages.mobile.page_provider import PageProvider as MobilePageProvider`